client/src/components/Home.js

```diff
@@ -86,7 +86,331 @@ function Details(props) {
   );
 }
 
+function EventItem(props) {
```

Comment:
This code should no longer be here, instead we use EventItem.js

Reply:
I transferred the funtion to the file